Resources/ServerInfo/Guidebook/_RMC14/RMCIntel.xml

```diff
@@ -0,0 +1,111 @@
+  \n\t- The bigger they are, the more points they award.
+  \n\t- Note that immature ones (Parasites, Larvas and Lesser) do not award any points.
+
+  [color=#A88B5E][bold]Documents and devices[/bold][/color]
+
+  <Box>
+    <GuideEntityEmbed Entity="RMCIntelPaperScrap" />
+    <GuideEntityEmbed Entity="RMCIntelProgressReport" />
+    <GuideEntityEmbed Entity="RMCIntelFolder" />
+    <GuideEntityEmbed Entity="RMCIntelTechnicalManual" />
+  </Box>
+
+  1. These documents must be read by trained personal.
+  \n\t\t\t- More complex documents require that you have read easier ones before.
+  \n\t\t\t- scrap -> report -> folder -> manual
+  \n2. The gained intel must then be entered into the intel computer found in the intel room.
+  \n3. The read document must then be stored in the intel room.
+
+  <Box>
+    <GuideEntityEmbed Entity="RMCIntelRetrieveHealthAnalyzer" />
+  </Box>
+
+  - These devices must be stored in the intel room.
+
+  ## Working in intelligence
+
+  <Box>
+    <GuideEntityEmbed Entity="CMGuidebookRoleASO" />
+    <GuideEntityEmbed Entity="CMGuidebookRoleIO" />
+    <GuideEntityEmbed Entity="CMGuidebookRoleYou" />
+  </Box>
+
+  You have chosen, or been chosen, to work in intelligence.
+  Your primary role will be to collect and process intel from the sources mentioned above.
+
+  <Box>
+    <GuideEntityEmbed Entity="RMCPouchDocument" />
+    <GuideEntityEmbed Entity="RMCIntelDetector" />
+    <GuideEntityEmbed Entity="RMCFulton" />
+    <GuideEntityEmbed Entity="CMEncryptionKeyIntel" />
+  </Box>
+
+  To aid you in this you have been supplied with the tools listed above.\nNote that [textlink="requisitions" link="RMCRequisitions"] also has these in stock.
+
+  <Box>
+    <GuideEntityEmbed Entity="RMCVisorNightVision" />
+    <GuideEntityEmbed Entity="RMCKitIntel" />
+  </Box>
+
+  There are however two "unique" things that requisitions can't help you with.
+  One is the rechargeable night vision optic that allow you to see in the dark for a limited amount of time.
+  The other is the field intelligence support kit that includes the tools listed above plus an intelligence pamphlet that teaches the reader how work the intel computer.
+
+  <Box>
+    <GuideEntityEmbed Entity="RMCComputerIntel" />
+    <GuideEntityEmbed Entity="RMCTechTreeConsole" />
+  </Box>
+
+  After getting geared up you should make yourself familiar with the devices in the intel room.
+  The intel computer is used to write down what you have read in documents and the tech tree console allows you to spend your collected intelligence.
+
+  [color=#A88B5E][bold]Deployment[/bold][/color]
+
+  Or rather: "[italic]Thanks for all that but how will it actually go down?[/italic]"
+
+  - The ASO will sit in the intel lab to read and type down all incoming documents.
+  - The CMO will drag dead xenos into the research lab.
+  - The Pilots and DCCs will install one fulton recovery system per dropship.
```

Comment:
```suggestion
  - The Pilot and DCC will install a fulton recovery system, normally on the Normandy, but sometimes also on the Alamo.
```

Reply:
There are multiple pilots and DCCs, so I will keep that as plural.